website/common/locales/en/groups.json

```diff
@@ -258,5 +258,9 @@
   "paymentDetails": "Payment Details",
   "aboutToJoinCancelledGroupPlan": "You are about to join a group with a canceled plan. You will NOT receive a free subscription.",
   "cannotChangeLeaderWithActiveGroupPlan": "You can not change the leader while the group has an active plan.",
-  "leaderCannotLeaveGroupWithActiveGroup": "A leader can not leave a group while the group has an active plan"
+  "leaderCannotLeaveGroupWithActiveGroup": "A leader can not leave a group while the group has an active plan",
+  "youHaveGroupPlan": "You have a free subscription because you are a member of a group that has a Group Plan. This will end when you  are no longer in the group that has a group plan. Any extra months you have will be applied at the end of the group plan.",
```

Comment:
Can the last sentence be: "Any months of extra subscription credit you have will be applied at the end of the group plan."
That's consistent with the terminology used in the extra months description.